examples/datasets/plot_spm_faces_dataset.py

```diff
@@ -97,8 +98,12 @@
 # Compute forward model
 
 # Make source space
-src = mne.setup_source_space('spm', spacing='oct6', subjects_dir=subjects_dir,
-                             overwrite=True)
+src_fname = data_path + '/subjects/spm/bem/spm-oct-6-src.fif'
+if not os.path.isfile(src_fname):
+    src = mne.setup_source_space('spm', src_fname, spacing='oct6',
+                                 subjects_dir=subjects_dir, overwrite=True)
+else:
+    src = mne.read_source_spaces(src_fname)
```

Comment:
Also snuck in this few-line edit that speeds up our `plot_spm_faces_dataset.py` example